source/scss/_pf-component-alert.scss

```diff
@@ -0,0 +1,38 @@
+//
+// Alerts
+// --------------------------------------------------
+
+.alert {
+  padding-left: $spacer;
+  padding-right: $spacer;
+  display: flex;
+  align-items: center;
+
+  .alert-link {
```

Comment:
It comes nested from patternfly. So I could unnest if I create a specific class. Do u think it worth it?